apps/worker/.env.local.example

```
@@ -1,6 +1,9 @@
 NODE_ENV=development
-# Google Generative AI
+# Google Generative AI (embeddings only)
 GOOGLE_GENERATIVE_AI_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_GENERATIVE_AI_API_KEY key should go before the NODE_ENV key

(UnorderedKey)